src/requires.jl

```diff
@@ -15,9 +15,11 @@ function __init__()
     end)
 
     @require Colors="5ae59095-9a9b-59fe-a467-6f913c188581" begin
+        fixnotation(c::Float64) = round(c, digits = 4)
         function PGFPlotsX.print_opt(io::IO, c::Colors.Colorant)
             rgb = convert(Colors.RGB, c)
-            rgb_64 = convert.(Float64, (rgb.r, rgb.g, rgb.b))
+            _rgb_64 = convert.(Float64, (rgb.r, rgb.g, rgb.b))
```

Comment:
Instead of repeating these 3 lines 3 times, `fixnotation` could be renamed to `_rgb_for_printing` or something like that, and take care of converting and rounding all 3 components. Maybe this function should be defined in the `PGFPlotsX` namespace.

Eg
```julia
function PGFPlotsX._rgb_for_printing(c::Colors.Colorant)
    rgb = convert(Colors.RGB, c)
    round.(Float64.(rgb.r, rgb.g, rgb.b); digits = 4)
end
```

Reply:
Sounds great